app/importers/companies_import.rb

```diff
@@ -0,0 +1,68 @@
+class CompaniesImport
+  include ActiveModel::Model
+  require 'roo'
+
+  AMOUNT_ITEMS_IN_ROW = 3
+  START_ROW = 2
+
+  attr_accessor :file
+
+  def initialize(attributes = {})
+    attributes.each { |name, value| send("#{name}=", value) }
+  end
+
+  def save
+    if find_errors(items).map { |c| c.errors.messages.empty? }.all?
```

Comment:
select {|c| c.errors.messages.empty? }.empty? may be better

Reply:
select return array of hashes and `[{},{}...].empty?` or `[{},{e: 'd'}...]` will be `false`.